Test failures with libspatialindex 2.0.0

The Debian package build fails due to test failures when using spatialindex 2.0.0 from experimental:
```
I: pybuild base:311: cd /build/python-rtree-1.2.0/.pybuild/cpython3_3.12_rtree/build; python3.12 -m pytest tests
============================= test session starts ==============================
platform linux -- Python 3.12.3, pytest-8.2.1, pluggy-1.5.0
rootdir: /build/python-rtree-1.2.0/.pybuild/cpython3_3.12_rtree/build
configfile: pyproject.toml
collected 40 items

tests/test_finder.py ..                                                  [  5%]
tests/test_index.py F.......x...........Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
.................                [ 97%]
tests/test_tpr.py .                                                      [100%]

=================================== FAILURES ===================================
______________________ IndexVersion.test_libsidx_version _______________________

self = <tests.test_index.IndexVersion testMethod=test_libsidx_version>

    def test_libsidx_version(self) -> None:
>       self.assertTrue(index.major_version == 1)
E       AssertionError: False is not true

/build/python-rtree-1.2.0/.pybuild/cpython3_3.12_rtree/build/tests/test_index.py:47: AssertionError
=========================== short test summary info ============================
FAILED tests/test_index.py::IndexVersion::test_libsidx_version - AssertionErr...
=================== 1 failed, 38 passed, 1 xfailed in 0.30s ====================
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
E: pybuild pybuild:389: test: plugin pyproject failed with: exit code=1: cd /build/python-rtree-1.2.0/.pybuild/cpython3_3.12_rtree/build; python3.12 -m pytest tests
I: pybuild base:311: cd /build/python-rtree-1.2.0/.pybuild/cpython3_3.11_rtree/build; python3.11 -m pytest tests
============================= test session starts ==============================
platform linux -- Python 3.11.9, pytest-8.2.1, pluggy-1.5.0
rootdir: /build/python-rtree-1.2.0/.pybuild/cpython3_3.11_rtree/build
configfile: pyproject.toml
collected 40 items

tests/test_finder.py ..                                                  [  5%]
tests/test_index.py F.......x...........Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
.................                [ 97%]
tests/test_tpr.py .                                                      [100%]

=================================== FAILURES ===================================
______________________ IndexVersion.test_libsidx_version _______________________

self = <tests.test_index.IndexVersion testMethod=test_libsidx_version>

    def test_libsidx_version(self) -> None:
>       self.assertTrue(index.major_version == 1)
E       AssertionError: False is not true

/build/python-rtree-1.2.0/.pybuild/cpython3_3.11_rtree/build/tests/test_index.py:47: AssertionError
=========================== short test summary info ============================
FAILED tests/test_index.py::IndexVersion::test_libsidx_version - AssertionErr...
=================== 1 failed, 38 passed, 1 xfailed in 0.31s ====================
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
Lost pointers: 0
E: pybuild pybuild:389: test: plugin pyproject failed with: exit code=1: cd /build/python-rtree-1.2.0/.pybuild/cpython3_3.11_rtree/build; python3.11 -m pytest tests
```

This patch may suffice:
```diff
--- a/tests/test_index.py
+++ b/tests/test_index.py
@@ -44,8 +44,8 @@ class IndexTestCase(unittest.TestCase):
 
 class IndexVersion(unittest.TestCase):
     def test_libsidx_version(self) -> None:
-        self.assertTrue(index.major_version == 1)
-        self.assertTrue(index.minor_version >= 7)
+        self.assertTrue(index.major_version in [1, 2])
+        self.assertTrue((index.major_version == 1 and index.minor_version >= 7) or index.major_version > 1)
 
 
 class IndexCount(unittest.TestCase):
```

--- a/tests/test_index.py
+++ b/tests/test_index.py
@@ -42,12 +42,6 @@ class IndexTestCase(unittest.TestCase):
         assert next(rtree_test.intersection([1, 1, 1, 2, 2, 2])) == 1564
 
 
-class IndexVersion(unittest.TestCase):
-    def test_libsidx_version(self) -> None:
-        self.assertTrue(index.major_version == 1)
-        self.assertTrue(index.minor_version >= 7)
-
-
 class IndexCount(unittest.TestCase):
     def setUp(self) -> None:
         self.boxes15 = np.genfromtxt("boxes_15x15.data")

